others/make_deb

```diff
@@ -127,14 +127,23 @@ if [[ ${branch_name} = "main" ]] || [[ ${branch_name} =~ ^[0-9]+\.[0-9]+\.[0-9]+
 # Debian file structure
 mkdir -p usr/local/lib
 if [ "${ADDITIONAL_LIB_NAME}" == "mqtt" ];  then
-    cp -R --preserve=links /usr/local/lib/libpaho* usr/local/lib
+    # For Raspberry Pi OS Bullseye
+    if [[ ${os_name} == *"Raspbian"* && ${os_version} == *"11"* ]]; then
```

Comment:
Instead of NAME, We need to rely on VERSION_CODENAME for bullseye check; Given different information for armv7l vs aarch64